gradoop-common/src/main/java/org/gradoop/common/model/impl/properties/PropertyValueList.java

```diff
@@ -179,7 +180,7 @@ public boolean hasNext() {
         try {
           inputStream.close();
         } catch (IOException e) {
-          e.printStackTrace();
+          throw new UncheckedIOException(e);
```

Comment:
This changes the semantics of this function. I think that the `hasNext` function should just ignore this exception.

Reply:
I agree ... if closing the stream fails, is not necessary to exit the application. @timo95 Please change your comment to something like `Ignore this exception because it is not critical in this case`